README.md

```diff
@@ -36,62 +36,38 @@ The [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscja
-
-### Getting Started for JUnit 4
-Please refer to [Download and Install](https://github.com/junit-team/junit4/wiki/Download-and-Install) from the JUnit 4's official document for the getting started guide.
-
-### Getting Started for TestNG
-
-Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's official document for getting started guide.
-
 ## Features
 
 ### Run/Debug Test Cases
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/run_codelens.png" style="border-radius: 15px" alt="Run Code Lens"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/editor-decoration.png" alt="Run/Debug Test Cases"/>
 </p>
 
-- The extension will generate `Run Test` and `Debug Test` shortcuts (also known as Code Lens) above the class and method definition. Simply click on them will start running or debugging the target test cases.
-
-> Note: If you cannot see the Code Lens in your editor, please refer to this [issue comment](https://github.com/Microsoft/vscode-java-test/issues/470#issuecomment-444681714) as a workaround.
+- The extension will generate shortcuts (the green play button) at the left side of the class and method definition. Simply click on it will start running the target test cases. Or you can right click on it to see more options.
 
 ---
 
 ### Test Explorer
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/run_explorer.png" style="border-radius: 15px" alt="Run Explorer"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/test_explorer.png" alt="Test Explorer"/>
 </p>
 
-- The Test Explorer is the place to show all the test cases in your project. You can also run/debug your test cases from here.
-- Click the node in the Test Explorer will navigate to the location of the source code.
-
-> Note: If the Test Explorer is empty, please refer to this [issue comment](https://github.com/Microsoft/vscode-java-test/issues/470#issuecomment-444681714) as a workaround.
+- The Test Explorer is the place to show all the test cases in your workspace. You can also run/debug your test cases from here.
 
 ---
 
 ### Customize Test Configurations
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/configuration.png" style="border-radius: 15px" alt="Customize Test Configurations"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/configuration.png" alt="Customize Test Configurations"/>
 </p>
 
 - Sometimes you may want to customize the configuration for running the test cases. To achieve this, you can add it into your workspace settings under the section: `java.test.config`.
```

Comment:
```suggestion
- Sometimes you may want to customize the configuration of running the test cases. To achieve this, you can add the configuration into your workspace settings under the section: `java.test.config`.
```

Reply:
How about 'you may want to customize the configuration **to run** the test cases'